src/core/Akka.Tests/Dispatch/ActorMailboxSpec.cs

```diff
@@ -0,0 +1,177 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MailboxesSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
```

Comment:
Could you change the name from Typesafe to Lightbend?
I already created another PR which does that for all other files.